include/lsst/meas/algorithms/KernelPsfFactory.h

```diff
@@ -13,6 +13,7 @@
 #include "lsst/afw/table/io/InputArchive.h"
 #include "lsst/afw/table/io/OutputArchive.h"
 #include "lsst/afw/table/io/CatalogVector.h"
+#include "lsst/afw/table/aggregates.h"
```

Comment:
This is still probably only 50% of the code in table.h, and we certainly don't want all of that.  We could add InputArchiive.h and OutputArchive.h to CatalogVector.h, as any header that uses the latter will always want the former, but I figured that was going against the spirit of the consensus of RFC-52 (even though that's about EUPS dependencies, not headers).

In any case, in general we should absolutely not be using the full-package headers anywhere in our C++ code - that's the path to unnecessary code bloat.  IMO those should only be used in Swig files, where we're essentially forced to import all headers in a package, and if we someday address that problem we should remove them.